vowpalwabbit/parse_args.cc

```diff
@@ -1529,7 +1531,23 @@ void finish(vw& all, bool delete_all)
 
     all.trace_message << endl << "total feature number = " << all.sd->total_features;
     if (all.sd->queries > 0)
-      all.trace_message << endl << "total queries = " << all.sd->queries << endl;
+      all.trace_message << endl << "total queries = " << all.sd->queries;
+    if (all.sd->overlapped_and_range_small > 0)
```

Comment:
All this codes seems like it should live in a finish() call in the reduction itself rather than being globally invoked.